md/src/main/java/org/cogcomp/md/BIOReader.java

```diff
@@ -155,15 +158,14 @@ private void annotateTas(){
             bcsl.add(false);
             bcsl.add(false);
             bcsl.add(false);
-            BrownClusters.init(bcs, bcst, bcsl, false);
+            brownClusters = BrownClusters.get(bcs, bcst, bcsl);
             WordNetManager.loadConfigAsClasspathResource(true);
             wordNet = WordNetManager.getInstance();
         }
         catch (Exception e){
             e.printStackTrace();
```

Comment:
I agree completely, however, this is not my code, it's nothing that I have changed, it has been this way. I can throw the exceptions up, but the exception apparently end up being thrown in the reader generation code inside the LBJava stuff, and again, I don't even know what this code does. So I can make the change, I can throw the exception up to the reader, but I can't change the getReader method signature of the LBJ generated code, or at least I shouldn't. I propose we throw the exceptions, but in the constructor for this reader, catch the thrown exceptions, convert them to a runtime exception and throw that so we don't have to change the get reader method. And the author should be made aware that this sort of absence of thought about error handling is wrong minded and unacceptable.

Reply:
I realize this is not your coding practice, just saw it in the context of your changes. I think your proposal is good. If you think it will take too much time right now, you can skip making the change, but open an issue that describes them.

Reply:
I have made the change, and from just a cursor glance, I think it will work out just fine.